.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Documentation:
+  Enabled: false
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
Performance is surely the last reason I would consider, given the choice of single vs. double quotes. For me it really is primarily about conveying _intention_, and having a couple additional tools at your disposal for quickly scanning code, or grepping.